src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -631,6 +634,7 @@ class LicenceService(
     licenceRepository.saveAndFlush(updatedLicenceEntity)
   }
 
+  @Transactional
```

Comment:
this was the only thing that actually does multiple updates and was called directly by a controller